assessments/0002-notary-project.md

```diff
@@ -0,0 +1,195 @@
+
+### Comments & recommendations
+
+- Most of the issues in this section are a function of not having a website yet, and the project still being in a building phase.
+  - As the website build process starts, put together a skeleton IA that can be filled in as the project grows.
+
+- There are documented community communication methods, but it could be collected and expanded during the site development process. Prometheus has a good example of a community page: https://prometheus.io/community/. 
+
+- Regarding beginner friendliness, consider processes/documents for:
+   - Triaging docs issues
+   - Clearly marking a way for new contributors to make code or documentation contributions (i.e. a “good first issue” label), and define what makes a good first issue
+   - Set up a process for maintaining/pruning stale issues
+   - New contributors/your first contribution
+   - Letting new users know where to get help
+
+- Complete the governance work started here: https://github.com/notaryproject/notaryproject/issues/55 https://github.com/notaryproject/notaryproject/pull/78, and add it to a governance page (or section) on the website.
+
+
+## Final Recommendations
+
+Is it possible plan documentation along side the V2 development? Will the documentation be able to get its structure from the specifications being developed?
+
+### Website
+
+- [x] Create separate documentation repository: https://github.com/notaryproject/notaryproject.dev
+- [ ] Setup a [Hugo website](https://gohugo.io) with a [Docsy theme](https://www.docsy.dev/)
+- [ ] Setup Netlify for hosting/CI
+- [ ] Develop a [maintainers list](https://github.com/notaryproject/notaryproject.dev/blob/main/.github/settings.yml) for the documentation and its repository (this has been started https://github.com/notaryproject/notaryproject/issues/77)
+- [ ] Develop a maintenance plan for the documentation and its repository 
+- [ ] Follow the [CNCF website guidelines checklist](https://github.com/cncf/techdocs/blob/main/howto/website-guidelines-checklist.md) for other required elements
+
+_Note_: [Notary Project branding artwork](https://github.com/cncf/artwork/tree/master/projects/notary) exists. Other branding elements, like colour schemes and whatnot, will need to be developed.
+
+### Information Architecture
+
+The content to be written could have the following structure (as a start - modifications will need to be made as development continues):
+
+**Site structure**
+
+- Landing page
+- Community
+   - Governance
+   - Community (details)
+   - Contributing
+      - Where to start
+      - How we work
+      - Filing and working on issues for the project
+      - Where the community is (contacts, meetings)
+      - Contributing to docs
+      - Where to ask for more help contributing
+      - Next steps
+- About
+- Blog (Get the word out about the project and how it's progressing)
+- Docs
+   - This section to be planned with the Notary Maintainers team
+   - Getting Started
```

Comment:
I had considered that -- but I was thinking ahead to any potential versioning. I think a Getting Started guide should be organized within the versioned portion of a site as it could change with project updates. A Getting Started guide within the docs folder can (and should) still be referenced on the home page, and is in my **Top Nav** suggestion below.

Reply:
For now let's leave it and see how it plays out in the final website.